pytorch_lightning/metrics/functional/sequence.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+try:
+    from torchtext.data.metrics import bleu_score as bleu
+except ImportError:  # pragma: no-cover
+    _TORCHTEXT_AVAILABLE = False
+else:
```

Comment:
@Borda why should this be in `else`? Can't we do this after the import from torchtext? IMO this would be cleaner and easier to read

Reply:
the else separate what is the code block potential to fail and what the actions according to the results